modules/ROOT/pages/dw-core-functions-min.adoc

```diff
@@ -8,11 +8,10 @@ endif::[]
 [[min1]]
 == min&#40;Array<T&#62;&#41;: T &#124; Null
 
-Returns the lowest numeric value in an array.
+Returns the lowest `Comparable` value in an array.
 
 
-Returns null if the array is empty and produces an error when non-numeric
-values are in the array.
+The items need to be of the same type it will throw an error if they are not, and it returns null when the array is empty.
```

Comment:
same as previous comment

Reply:
done in adoc file only, will update dw repo separately

Reply:
done in dw